strings.txt

```diff
@@ -7001,6 +7006,7 @@
     comment = About short text (below logo)
     tags = ios, android
     en = Joined efforts of Mail.Ru Group and OpenStreetMap make the most useful application for travelers!
+    fr = L'application la plus utile pour les voyageurs, a été créé avec la participation des équipes Mail.Ru et OpenStreetMap.
```

Comment:
There will be a merge conflict with https://github.com/mapsme/omim/pull/4159/files#diff-744cbd21eb2f75a5b888f993234e8898R19424